src/ape_console/_cli.py

```diff
@@ -127,4 +123,11 @@ def console(project=None, verbose=None, extra_locals=None):
     if console_extras:
         namespace.update(console_extras)
 
-    IPython.embed(colors="Neutral", banner1=banner, user_ns=namespace)
+    from traitlets.config import Config
+
+    config = Config()
```

Comment:
Is it safe to assume an empty config when `not environ.get("APE_TESTING")` or should I have two separate paths and not provide the `config` kwarg at all if the env var isn't set?